dataland-frontend/tests/e2e/fixtures/CompanyFixtures.ts

```diff
@@ -30,6 +30,15 @@ function getRandomIdentifiers(): { [p: string]: string[] } {
   return identifiers;
 }
 
+/**
+ * Generates a random company contact email address
+ * @returns a random company contact email address
+ */
+function generateCompanyContactDetails(): string {
+  const randomFourDigitNumber = faker.number.int({ min: 1, max: 999 });
+  return `dummy${randomFourDigitNumber}@email.com`;
+}
+
```

Comment:
Ich glaube du brauchst hier keine custom Funktion um email Addressen zu generieren.

Du kannst einfach `faker.internet.email()` nutzen.

Reply:
Schau mal im File `StoredDataRequestMessageFaker.ts`, da wird dieser faker auch benutzt, um Email Adressen zu faken.

Reply:
updated it

Reply:
Ich sehe keine Änderung